app/src/androidTest/java/org/mozilla/fenix/ui/robots/MultipleSelectionToolbarRobot.kt

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Because we're going to eventually have a LOT of robots and because they all have to live in the same flat directory, to the extent possible, I find that it's useful to name robots that are children of another UI in such a way that they group together alphabetically.  Take a look at the Settings submenu robots, for example (https://github.com/mozilla-mobile/fenix/tree/master/app/src/androidTest/java/org/mozilla/fenix/ui/robots).   It might take some creativity to prevent the names from getting too long, but at least if you can indicate the parent UI that would be better.  Perhaps something like "HistoryMultiSelectToolbarRobot" might work.

Reply:
renamed.